cookbook/core/containerization/multi_images.py

```diff
@@ -46,13 +46,102 @@
 
+If you wish to build and push your Docker image to Dockerhub through your account, follow the below steps:
+
+1. Create an account with `Dockerhub <https://hub.docker.com/signup>`__.
+2. Build a Docker image using the Dockerfile:
+    
+.. code-block::
+
+   docker build . -f ./<dockerfile-folder>/<dockerfile-name> -t <your-name>/<docker-image-name>:<version>
+3. Once the Docker image is built, login to your Dockerhub account from the CLI:
+    
+.. code-block::
+
+   docker login
+4. It will prompt you to enter the username and the password.
+5. Push the Docker image to Dockerhub:
+    
+.. code-block::
+        
+   docker push <your-dockerhub-name>/<docker-image-name>
+
+Example: Suppose your Dockerfile is named `Dockerfile_prediction`, Docker image name is `multi-images-prediction` with the `latest` version, your build and push commands would look like:
+
+.. code-block::
+
+   docker build -f ./path-to-dockerfile/Dockerfile_prediction -t username/multi-images-prediction:latest
+   docker login
+   docker push dockerhub_name/multi-images-prediction
+
+
+
+Let us understand how multiple images can be used within a single workflow using an example.
 """
+# %%
+# Import the necessary dependencies. 
+from flytekit import task, workflow
+import pandas as pd
+from typing import NamedTuple
+from sklearn.model_selection import train_test_split
+from sklearn.svm import SVC
+
+split_data = NamedTuple(
+    "split_data",
+    train_features=pd.DataFrame,
+    test_features=pd.DataFrame,
+    train_labels=pd.DataFrame,
+    test_labels=pd.DataFrame,
+)
+
+# %%
+# Define a task that fetches data and splits the data into train and test sets.
+@task(container_image="ghcr.io/flyteorg/flytecookbook:multi-image-predict-adc4aaf34cacc293dd508da03f2839f50a484367b597abf1c8163625cc2b674d")
+def svm_trainer() -> split_data:
+    dataset_url = "https://raw.githubusercontent.com/harika-bonthu/SupportVectorClassifier/main/datasets_229906_491820_Fish.csv"
+    fish_data = pd.read_csv(dataset_url)   
+    X = fish_data.drop(['Species'], axis = 'columns')
+    y = fish_data.Species
+    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size= 0.31)
+    y_train = y_train.to_frame()
+    y_test = y_test.to_frame()
+
+    return split_data(
+        train_features=X_train,
+        test_features=X_test,
+        train_labels=y_train,
+        test_labels=y_test,
+    )
+
+# %%
+# .. note ::
+#     To use your own Docker image, replace the value of `container_image` with the fully qualified name that identifies where the image was pushed.
+
+# %%
+# Define another task that trains the model on the data and computes the accuracy score.
+@task(container_image="ghcr.io/flyteorg/flytecookbook:multi-images-preprocess-3edc8311755363c271acc9ed1ec733b4caf21ea1e387782214acff1510a08d58") 
+def svm_predictor(X_train:pd.DataFrame,X_test:pd.DataFrame,y_train:pd.DataFrame,y_test:pd.DataFrame) -> float:
```

Comment:
nit: spacing `X_train: pd.DataFrame, X_test: pd.DataFrame`, etc.

for readability, indent in new line:
```
def svm_predictor(
    X_train: pd.DataFrame,
    ...
) -> float:
```